server/app/routes/v1/services/service_container_logs.rb

```diff
@@ -5,7 +5,7 @@
     r.is do
       scope = @grid_service.container_logs
 
-      scope = scope.where(name: r['container']) unless r['container'].nil?
```

Comment:
Just thinking what is the correct behaviour when both instance and container are present?